glossary.md

```diff
@@ -56,7 +58,7 @@ A set of rules for a specific Game.
 
 ## Game \(Validated\)
 
-Game has to have more than 2 players per team at the end. So don't leave the game until next map starts, or use `/mastermode 2` \(match mode\).
+Game has to have more than 2 players per team at the end. So don't leave the game until next map starts, or use `/mastermode 2` \(match mode\). Don't change nickname \(shouldn't happen with authenticated users\).
```

Comment:
Don't think you need to escape brackets `\(`.

Also try to put new sentences on new Markdown lines. This will still display as a single line but be easier to version on Git.